Rakefile

```diff
@@ -3,10 +3,21 @@
 require "bundler/gem_tasks"
 require "rake/testtask"
 
-begin
-  require 'rspec/core/rake_task'
-  RSpec::Core::RakeTask.new(:spec)
-rescue LoadError # rubocop:disable Lint/SuppressedException
+Rake.application.options.trace = false
+
+Rake::TestTask.new do |t|
+  t.libs << "lib"
+  t.libs << "spec"
+  t.warning = false
+  t.test_files = FileList['spec/**/*_spec.rb']
+end
+
+task(:spec) do
+  begin
+    Rake::Task[:test].execute
+  rescue RuntimeError
+    exit(1)
+  end
 end
```

Comment:
I added this block for a weird reason. By default, if the Rake test task fails, Rake ends up printing an error message about the task having exited with a non-zero exit status and also prints a backtrace, which I found confusing and unnecessarily verbose.

So, I have a task here that executes the other task, catches the runtime error and exits with a non-zero status to signal failure. That prevents the Rake error message/backtrace from being printed in the console.